torch/csrc/jit/serialization/export.cpp

```diff
@@ -102,6 +102,7 @@ void validateBlock(
             "\n\nDefined at:\n" + getNodeStackTraceString(node))
       }
     } else {
+#ifdef BUILD_CAFFE2
```

Comment:
Nothing in this code block mentions Caffe2. Could you add a comment to make it clear why we think this is caffe2 specific?

Reply:
Sure. I *assumed* this is a Caffe2-specific change because this block modifies an `aten::expand` only when `operator_export_type == onnx_torch::OperatorExportTypes::ONNX_ATEN_FALLBACK` but not when `operator_export_type == onnx_torch::OperatorExportTypes::ONNX_ATEN`.

All previous changes conditioned to the aten fallback operator export type were meant to Caffe2-only scenarios